website/docs/reference/integrations/slack-app.md

```diff
@@ -0,0 +1,64 @@
+---
+id: slack-app
```

Comment:
When the id is the same as the file name, you don't need the id. In fact, most of our files don't have this, so I'd suggest removing it for consistency.
```suggestion
```